src/style.css

```diff
@@ -0,0 +1,119 @@
+html {
+  font-family: "Roboto", sans-serif;
+
+  --small-text: 11px;
+  --blue-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.product-card {
+  display: flex;
+  width: 200px;
+  margin: 0;
+  padding: 14px;
+  flex-wrap: wrap;
+  justify-content: center;
+  border: 1px solid #f3f3f3;
+}
+
+.product-card:hover {
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+}
+
+.product-card__img {
+  margin: 15px 20px 45px;
```

Comment:
![image](https://user-images.githubusercontent.com/35385361/113323932-32c56a80-931f-11eb-913e-ecbc7c9fc214.png)
15px and 20px it's too many

Reply:
you already have padding why you need a horizontal margin?